src/Http/Http.Abstractions/src/HttpRequest.cs

```diff
@@ -99,13 +99,13 @@ public abstract class HttpRequest
         public abstract string ContentType { get; set; }
 
         /// <summary>
-        /// Gets or sets the RequestBody Stream.
+        /// Gets or sets the request body <see cref="Stream"/>.
         /// </summary>
-        /// <returns>The RequestBody Stream.</returns>
+        /// <returns>The request body <see cref="Stream"/>.</returns>
```

Comment:
I think properties should be using `<value>` rather than `<returns>`. For example: https://docs.microsoft.com/en-us/dotnet/api/system.string.length?view=netframework-4.8#property-value

Reply:
```suggestion
        /// <value>The request body <see cref="Stream"/>.</value>
```

Reply:
Yea I was surprised they all did `<returns>` as it doesn't even show up in intellisense.